robottelo/ui/browser.py

```diff
@@ -109,6 +113,18 @@ def browser():
                 Ie() if settings.webdriver_binary is None
                 else Ie(executable_path=settings.webdriver_binary)
             )
+        elif webdriver_name == 'edge':
+            capabilities = webdriver.DesiredCapabilities.EDGE.copy()
+            capabilities['acceptSslCerts'] = True
```

Comment:
This line seems to have no effect currently 😞  But i'd still like to add it as it's much better way to avoid cert error rather then clicking the button each new session. Maybe in some newer Edge/WebDriver versions it will work as expected and will save us extra click per session.